### DISCUSSION TITLE
How do you configure a custom-run-coordinator in cloud

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18162] [Q&A]
Hello team! :man-raising-hand:
:questiongif: _*CustomRunCoordinator*_ &amp; _*Dagster.Cloud*_ :dagsir: question :questiongif:

I implemented a _*custom-run-coordinator*_ which works great locally on `dagster dev` and is configured like this in `dagster.yaml`
```run_coordinator:
  module: run_coordinator
  class: CustomRunCoordinator
  config:
    tag_concurrency_limits:
      - key: "my_tag"
        value:
          applyLimitPerUniqueValue: true
        limit: 1```
I explored almost every bit of the web, trying to figure out how to define my `CustomRunCoordinator` from `project-root/run_coordinator.py` as my dagster.cloud run coordinator, and still haven't figured it out.
How can I specify a concrete run coordinator for cloud deployments? (preferably via `dagster_cloud_staging.yaml` /`dagster_cloud_prod.yaml` or other config files and not through the UI/CLI)

Thank you very much, folks! :keanu-thanks: :pray:

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700417576735309?thread_ts=1700417576.735309&cid=C01U954MEER).*

**Comment by eliranshemtov** [comment]
According to reply on _[#dagster-cloud](https://dagster.slack.com/archives/C02LJ7G0LAZ)_ Slack channel:
it’s not possible to use a custom run coordinator with Dagster Cloud - as we override that functionality to use a bespoke Dagster Cloud coordinator.
[original response in Slack](https://dagster.slack.com/archives/C02LJ7G0LAZ/p1700697728468259?thread_ts=1700466696.217759&cid=C02LJ7G0LAZ)
